projects/pt1/python/torch_mlir/jit_ir_importer/build_tools/abstract_interp_lib_gen.py

```diff
@@ -617,9 +617,82 @@ def aten〇_unsafe_view〡shape(self: List[int], size: List[int]) -> List[int]:
 def aten〇resize_〡shape(self: List[int], size: List[int], memory_format: Optional[int] = None) -> List[int]:
     return size
 
+def _max_pool3d(
```

Comment:
This function has some complex logic. Can you add tests for it? See example:

https://github.com/llvm/torch-mlir/blob/9e2a2851898fbd83df28674e2561d52910f3e83e/projects/pt1/python/torch_mlir/jit_ir_importer/build_tools/abstract_interp_lib_gen.py#L454